src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java

```diff
@@ -56,23 +56,38 @@ public ReleaseDroid(final RepositoryGateway repositoryGateway, final ValidateUse
     // [impl->dsn~rd-creates-release-report~1]
     public void run(final UserInput userInput) {
         final Repository repository = this.repositoryGateway.getRepository(userInput);
-        validateUserInput(userInput, repository);
+        final List<PlatformName> platformNames = getPlatformNames(userInput, repository);
+        validatePlatformNames(platformNames);
+        validateUserInput(userInput);
```

Comment:
Maybe we can move this line to the beginning of the method?

Reply:
Tried this out, this makes testing more difficult. I will leave it be for now

Reply:
Okay, decided to move it as you suggested